storage/replica.go

```diff
@@ -191,16 +192,20 @@ func NewReplica(desc *roachpb.RangeDescriptor, store *Store) (*Replica, error) {
 		tsCache:     NewTimestampCache(store.Clock()),
 		sequence:    NewSequenceCache(desc.RangeID),
 		pendingCmds: map[cmdIDKey]*pendingCmd{},
+		RangeID:     desc.RangeID,
```

Comment:
This is where we set the RangeID to be a first class value on the replica struct.  Since this is the only place where it is written, we don't need to lock the replica when we want to fetch the rangeID.